package-res/resources/web/pentaho/type/_doc/spec/IPropertyTypeProto.jsdoc

```diff
@@ -27,6 +27,22 @@
  */
 
 /**
+ * A reference to the property type's base type, if any.
+ *
+ * Can only be specified on a root property of a complex type,
```

Comment:
Suggest:

The property type's base type can only be specified on a root property of a complex type,